src/Perlang.Common/Utils.cs

```diff
@@ -13,6 +15,12 @@ public static string Stringify(object @object)
             {
                 return "null";
             }
+            else if (@object is IConvertible convertible)
+            {
+                // The explicit IFormatProvider is required to ensure we use 123.45 format, regardless of
+                // host OS language/region settings. See #xxx for more details.
```

Comment:
**TODO**: create a proper issue for this and amend these comments, pointing the reader to the issue in question.

Reply:
#263